tests/WebAppCallsMicrosoftGraph/Startup.cs

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Mvc.Authorization;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Identity.Web;
+using Microsoft.Identity.Web.TokenCacheProviders.InMemory;
+using Microsoft.Identity.Web.UI;
+
+namespace WebAppCallsMicrosoftGraph
+{
+    public class Startup
+    {
+        public Startup(IConfiguration configuration)
+        {
+            Configuration = configuration;
+        }
+
+        public IConfiguration Configuration { get; }
+
+        // This method gets called by the runtime. Use this method to add services to the container.
+        public void ConfigureServices(IServiceCollection services)
+        {
+            services.AddSignIn(Configuration)
+                    .AddWebAppCallsProtectedWebApi(Configuration, new string[] { "user.read" })
+                    .AddInMemoryTokenCaches();
+
+            services.AddMicrosoftGraph(Configuration);
```

Comment:
@jmprieur , @jennyf19 We reviewed this proposal with the working team today and I'm channeling some feedback here. Since the AddMicrosoftGraph abstraction is introducing a dependency on the Graph SDK, we feel this implementation can stay out of the Microsoft.Identity.Web and the developer can use the AddSingleton pattern which is pretty standard in ASP.NET.

Reply:
thanks @navyasric